src/components/UI/Calculator.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import './Calculator.css';
+import calculate from '../logic/calculate';
+import CalculatorRow from './CalculatorRow';
+
+const tableData = [
+  ['AC', '+/-', '%', '÷'],
+  ['7', '8', '9', 'x'],
+  ['4', '5', '6', '-'],
+  ['1', '2', '3', '+'],
+];
+
+function Calculator() {
```

Comment:
- [ ] I can see you are doing a great job on the hooks. However, you are advised to follow the ES6 syntax which lets us use arrow functions when declaring our functions. This makes our code look more modern and readable. Kindly look into this also in the `CalculatorRow` component.